SOC/models/manna.py

```diff
@@ -37,16 +40,17 @@ def topple(self) -> bool:
 
         :rtype: bool
         """
-        return topple(self.values, self.visited, self.critical_value, self.BOUNDARY_SIZE)
+        return topple(self.values, self.visited, self.critical_value, self.model_variant, self.BOUNDARY_SIZE)
 
     def dissipate(self):
         """Does nothing, dissipation is handled by the added boundary strips"""
         pass
 
 _DEBUG = True
 
-@numba.njit
-def topple(values: np.ndarray, visited: np.ndarray, critical_value: int, boundary_size: int) -> bool:
+# @numba.njit
```

Comment:
Wykomentowałeś `numba.njit`, więc obawiam się zwalniania dla większych rozmiarów... przywracamy?
```suggestion
@numba.njit
```

Reply:
Jasne. (dopiero teraz popróbowałem z użyciem numby)